beacon-chain/core/state/state_transition.go

```diff
@@ -168,6 +169,21 @@ func ExecuteStateTransition(
 	return newState, nil
 }
 
+// ApplyBlockOperations describes the per block operations that happen on every slot.
```

Comment:
Can you expand on this? Is there a spec definition or summary you can provide? From a godocs perspect, this doesn't add any new information